### PR TITLE
Remove useless argument to AlertNotify.

### PR DESCRIPTION
**Pull request by deadalnix**
As per title.

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -163,9 +163,9 @@ extern size_t nCoinCacheUsage;
 extern CFeeRate minRelayTxFee;
 /** Absolute maximum transaction fee (in satoshis) used by wallet and mempool (rejects high fee in sendrawtransaction) */
 extern CTweak<CAmount> maxTxFee;
-extern bool fEnableReplacement;  // BU TODO is this RBF flag?
 /** If the tip is older than this (in seconds), the node is considered to be in initial block download. */
 extern int64_t nMaxTipAge;
+extern bool fEnableReplacement;  // BU TODO is this RBF flag?
```
Comment (sickpig) [inline src/main.h]: this change seems unrelated to this PR

**Review by sickpig — CHANGES_REQUESTED** [review]
unrelated code change in `main.h`

File: src/main.cpp [hunk 2]
```diff
@@ -1840,10 +1840,7 @@ static void AlertNotify(const std::string& strMessage, bool fThread)
     safeStatus = singleQuote+safeStatus+singleQuote;
     boost::replace_all(strCmd, "%s", safeStatus);
 
-    if (fThread)
-        boost::thread t(runCommand, strCmd); // thread runs free
-    else
-        runCommand(strCmd);
+    boost::thread t(runCommand, strCmd); // thread runs free
```
Comment (sickpig) [inline src/main.cpp:1843]: fix a trailing comment in a new PR?

File: src/main.h (hunk 1 above)
Comment (sickpig) [reply src/main.h]: @deadalnix  as a pointed out already this change is unrelated to this PR.

**Review by sickpig — CHANGES_REQUESTED** [review]

**Comment by gandrewstone** [comment]
no backport needed, AlertNotify is not in 1.0

**Pull request merged**